src/appleseed/renderer/modeling/surfaceshader/nprsurfaceshaderhelper.cpp

```diff
@@ -64,9 +64,11 @@ void NPRSurfaceShaderHelper::evaluate(
     const Material* material = shading_point.get_material();
     const ShaderGroup* sg = material->get_render_data().m_shader_group;
 
+#ifdef APPLESEED_WITH_SPECTRAL_COLOR
```

Comment:
A slightly better option is to remove the `#ifdef` and check if `get_mode()` returns something else than `Spectrum::RGB`.

Reply:
There is not Spectrum::Spectral when spectral colors are not enabled

Reply:
I know, that's why I'm suggesting to do

    if (shading_components.m_beauty.get_mode() != Spectrum::RGB)